src/knodetable.rs

```diff
@@ -44,7 +44,7 @@ pub struct KNodeTable<TId, TAddr> {
 }
 
 /// K-bucket - structure for keeping last nodes in Kademlia.
-struct KBucket<TId, TAddr> {
+pub struct KBucket<TId, TAddr> {
```

Comment:
Yes, otherwise Rust complains about this:

```
error[E0446]: private type in public interface
  --> src/knodetable.rs:73:35
   |
73 |     pub fn buckets(&self) -> &Vec<KBucket<TId, TAddr>> {
   |                                   ^^^^^^^^^^^^^^^^^^^ can't leak private type

error: aborting due to previous error
```